addon/components/universal-selection/component.js

```diff
@@ -25,19 +25,29 @@ export default Component.extend({
     return this.get('exports').searchEntities(this.keyword).then((response) => {
       resolve(
         Object.keys(response).reduce((acc, entityType) => {
+          if ((response[entityType].length > 0) && !acc.find((item) => acc.includes(item.type))) {
+            acc.push({
+              groupName: entityType.charAt(0).toUpperCase() + entityType.slice(1),
```

Comment:
Use `capitalize` from @ember/string (`import { capitalize } from '@ember/string';`)

Reply:
ok!

Reply:
Actually, this is a display concern and should be handled in the template and not in the js. there is a `text-style-capitalize` class you can use

Reply:
Also, you are already building a group, so you don't need `group.groupName`, just `group.name`

Reply:
They are using `groupName` in the docs though (https://ember-power-select.com/docs/groups/)